build_docker.py

```diff
@@ -8,9 +8,19 @@
                                                  "Mantid")
     parser.add_argument("--image_name", type=str, default="scipp_vs_mantid",
                         help="the name of the image to start")
+    parser.add_argument("--http_proxy", action="store_true",
+                        help="use this address as http proxy.")
+    parser.add_argument("--https_proxy", action="store_true",
+                        help="use this address as http proxy.")
 
     args = parser.parse_args()
     docker_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
                                "Docker")
-    command = "docker build -t {} {}".format(args.image_name, docker_path)
+    command = "docker build -t {}".format(args.image_name)
+    if args.http_proxy:
+        command = "{} --build-arg http_proxy".format(command)
+    if args.https_proxy:
+        command = "{} --build-arg https_proxy".format(command, args.https_proxy)
```

Comment:
Are there some `{}` missing, and/or are there too many / too few arguments?

Reply:
No. Everything is correct `--http(s)_proxy` are the flags that forwards corresponding env variables.